docs/_templates/external-data-v1.rst

```diff
@@ -0,0 +1,381 @@
+                 :class: citation-table-citation citation-access-date
+
+                 {{ Data.citations.data_citation }}
+          {%- endif %}
+          {% if Data.citations.paper_citation %}
+          * - **Paper citation**
+            - .. code-block:: text
+                 :class: citation-table-citation
+
+                 {{ Data.citations.paper_citation }}
+          {%- endif %}
+          {% for citation in valid_custom_citations %}
+          * - **{{ citation.name }}**
+            - .. code-block:: text
+                 :class: citation-table-citation
+
+                 {{ citation.citation }}
+          {% endfor %}
+       {%- endif %}
+       {%- endif %}
+
+       .. {%- if valid_tags %}
+       .. .. tags:: {{ valid_tags | join(", ") }}
+       .. {%- endif %}
+
+    {% endif %}
+
+    {% if Data.enable_specifications %}
+    .. tab-item:: Specifications
+       :name: specifications
+
+       .. raw:: html
+
+          <div class="product-tab-table-of-contents"></div>
+
+       {% if valid_bands %}
+       .. rubric:: Bands
+          :name: bands
+          :class: h2
+
+       Bands are distinct layers of data within a product that can be loaded using the Open Data Cube (on the `DEA Sandbox <dea_sandbox_>`_ or `NCI <nci_>`_) or DEA's `STAC API <stac_api_>`_.
+
+       .. _dea_sandbox: https://knowledge.dea.ga.gov.au/guides/setup/Sandbox/sandbox/
+       .. _nci: https://knowledge.dea.ga.gov.au/guides/setup/NCI/basics/
+       .. _stac_api: https://knowledge.dea.ga.gov.au/guides/setup/gis/stac/
+
+       .. list-table::
+          :header-rows: 1
+
+          * - 
+            - Aliases
+            - Resolution
+            - CRS
+            - Nodata
+            - Type
+            - Description
+          {% for band in valid_bands %}
+          * - **{{ band.name }}**
+            - {{ band.aliases|join(', ') if band.aliases else none_text }}
+            - {{ band.resolution or not_available_text }}
+            - {{ band.crs or not_available_text }}
+            - {{ band.nodata }}
+            - {{ band.type or not_available_text }}
+            - {{ band.description or none_text }}
+          {% endfor %}
+
+       {{ Specifications.bands.footnotes if Specifications.bands.footnotes }}
+       {% endif %}
+    {% endif %}
+
+    {% if Data.enable_access %}
+    .. tab-item:: Access
+       :name: access
+
+       .. raw:: html
+
+          <div class="product-tab-table-of-contents"></div>
+
+       .. rubric:: Access the data
+          :name: access-the-data-2
+          :class: h2
+
+       {% if has_access_data %}
+       .. list-table::
+          :name: access-table
+
+          {% if valid_external_data %}
+          * - **{{ external_data_label }}**
+            - 
+              * `{{ valid_external_data.custom_label or external_data_default_name }} <{{ valid_external_data.link }}>`_
+            - The external page should contain information about how to access the data.
```

Comment:
Access the the external pages is probably more about users being able to learn more about the data vs. purely data access (given we're providing our own data access options via DEA's systems). Some possible alternative wordings:

```suggestion
            - Access additional information about this dataset from the external provider.
```

```suggestion
            - Learn more about this data from the external provider.
```

Reply:
> Learn how to access the data from the external provider.

Reply:
The "how to access the data" is the bit we don't want to include - we want users to access data using DEA systems, and the external provider's page won't provide details on how to load the data in a way that's relevant to DEA.

The purpose of these links is to allow users to find out more information about the products from the provider, allowing us to make our listings minimal and not reproduce that material.

Reply:
Ok very good point -- I've now reworded it to

> Learn more about the data from the external provider.